python/ngraph/impl/__init__.py

```diff
@@ -20,17 +20,24 @@
 
 # flake8: noqa
 
+import os
 import sys
 import six
 
-# workaround to load the libngraph.so with RTLD_GLOBAL
-if six.PY3:
-    import os
-    flags = os.RTLD_NOW | os.RTLD_GLOBAL
+if sys.platform == 'win32':
+    # ngraph.dll is currently located 3 directories above
```

Comment:
The user may choose to have a custom installation. The word `currently` would be misleading.
```suggestion
    # ngraph.dll is installed 3 directories above by default
```

Reply:
It's a code comment for ng developers, not documentation for the users. Also if the user changes location of ngraph.dll this code will stop working.

Reply:
Let's document that too.
> If you're using a custom installation of nGraph, add the location of ngraph.dll to your system PATH.

This is the type of documentation that will be found by someone debugging their problems with an nGraph installation. That includes users. I have been this user of other libs many times.